rust-src/concordium_base/src/bulletproofs/range_proof.rs

```diff
@@ -185,6 +189,15 @@ pub fn prove<C: Curve, T: Rng>(
         transcript.append_message(b"Vj", &V_j.0);
         V_vec.push(V_j);
     }
+
```

Comment:
I suggest we add these publics on top of everything, i.e., even before the commitments Vj-s. 
In general, the best would be to add things in the following order:
1. setup parameters, e.g., group generators, commitment keys, genesis, etc
2. statement
3. proof transcript --- in the same order as it is in the underlying interactive protocol